index.html

```diff
@@ -38,18 +38,34 @@
 						</div>
 					</div>
 				</nav>
-		<div class="d-flex justify-content-between mt-5">
-			<img src="./images/Ellipse 11.png" alt="circle">
+		<div class="d-flex justify-content-between">
+			<div>
+			<img src="./images/triagle.png" alt="circle">
 		</div>
-		<div class="d-flex justify-content-between mt-5">
-			<img class="icons" src="./images/Ellipse 5.png" alt="circle">
-			<img class="icons" src="./images/Ellipse 4.png" alt="circle">
-			<img class="icons" src="./images/Ellipse 3.png" alt="circle">
-
+		<div class="d-flex  mt-5">
+			<img class="icons" src="./images/Rectangle.png" alt="circle">
+			<img src="./images/cycle.png" alt="">
+		</div>
+		<div>
+			<img class="icons" src="./images/half.png" alt="circle">
+			<img class="icons" src="./images/smallcycle.png" alt="circle">
```

Comment:
-Those images are there for decoration. I think it is better to use them as a background image.